### ISSUE TITLE
Issue with default audio and video filters

### ISSUE THREAD

**Issue by Anacondo** [porcino/Av1ador#18]
I found an issue related to the default audio and video filters. It looks as if you select and edit them in any way, then change to another video, when you come back to the old one the values are reset back to defaults, ignoring the previous modification:

![Av1ador_2023-12-18_16-23-37](https://github.com/porcino/Av1ador/assets/5350602/e46b3b23-223c-4d11-a5ec-144a828c06c6)

This is related to #12 as it's currently impossible to modify the default Audio and Video filters, as they will be reset back to defaults after being edited. I observed the same in Video with the Resize and Tonemap filters.

**Comment by porcino** [comment]
> I observed the same in Video with the Resize and Tonemap filters

This doesn't happens to me, actually when I encode, I edit both most of the time and the changes are saved

**Comment by Anacondo** [comment]
I haven't been able to reproduce the issue with the tonemap filter, so I must have remember wrong, sorry about that. However, I was able to reproduce the issue with the scaling filter:

https://github.com/porcino/Av1ador/assets/5350602/6178da4e-aaad-418e-8987-94c089702b8f

I just want to use spline36, does your filter use it by default?

Also, I've tried the latest nightly to test the issue with the audio filters and I'm still getting the same:

https://github.com/porcino/Av1ador/assets/5350602/cc371645-1374-494e-8b7f-a8d203405005

Am I doing something wrong?

**Comment by Anacondo** [comment]
Damn it, I deleted my last reply by mistake, here it is again:

I haven't been able to trigger the issue with the tonemap filter, I must have remembered it wrong, sorry about that. However, I was able to trigger the issue with the scale filter:

https://github.com/porcino/Av1ador/assets/5350602/ce84af02-d6d9-410a-a83b-b91212bb19a0

I just want to be able to use spline36, does your filter use it by default?

Also, I've tested the audio issue with the latest Nightly from 2h ago and I'm having the same issue:

https://github.com/porcino/Av1ador/assets/5350602/9215124d-5cc2-4974-95d4-b7901634c15c

Am I doing something wrong?
